rollup.config.js

```diff
@@ -1,27 +1,26 @@
-'use strict';
-
-var babel = require('rollup-plugin-babel');
+import babel from '@rollup/plugin-babel';
 // uglify handles only es5 code, so this also acts as smoke test against shipping es2015+ syntax
-var uglify = require('rollup-plugin-uglify').uglify;
-var pkg = require('./package.json');
+import { uglify } from "rollup-plugin-uglify";
+import pkg from './package.json';
```

Comment:
currently we produce builds on node `8+`,
using import in plain js files would break that.

I'd suggest to keep using require for the time being.

Reply:
Ah, I didn't realise. It might be worth indicating that in `package.json`: https://stackoverflow.com/a/29349773/12895058
This means the latest rollup >=v2 will not work. I've tried 1.32.1 and that works fine, so I'll step it down to that.
What version of 8 is sufficient? Is support for 8.0.0 required, or is something like 8.10.0 acceptable?

Also, `import` statements work in at least node 7 as far as I can tell from testing it locally.

Reply:
esmodules are still marked as experimental in node 14:
https://nodejs.org/api/esm.html#esm_ecmascript_modules

___

I think changing that is slightly out of scope for this pr, 
we'll have to discuss whether to keep node 8 support or not moving forward, 
but I guess this could be a topic for a different story...

Reply:
The main reason for the change was to conform with the recommended rollup config format, but if we're dropping it to 1.X then that's no longer the case anyway. I'll revert it.

I still need to know what the absolute minimum engine version is. If 8.3.0 is acceptable then only rollup has to be lowered. If it's 8.0.0, then a rollup plugin will also have to be lowered.